pkg/tracing/opentelemetry/opentelemetry_test.go

```diff
@@ -68,14 +71,24 @@ func TestTracing(t *testing.T) {
 	}))
 	t.Cleanup(collector.Close)
 
-	newTracing, err := tracing.NewTracing("", 0, &Config{
-		Insecure: true,
-		Address:  strings.TrimPrefix(collector.URL, "http://"),
-	})
+	tracingConfig := &static.Tracing{
+		ServiceName: "traefik",
+		SampleRate:  1.0,
+		OTLP: &opentelemetry.Config{
+			HTTP: &opentelemetry.HTTP{
+				Endpoint: strings.TrimPrefix(collector.URL, "http://"),
```

Comment:
would we not care about `https://`? I know it's a test maybe we should have a second test to make sure HTTPS works?